tests/runtime/dwarf

```diff
@@ -31,44 +31,44 @@ TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
 
 NAME uprobe arg by name - struct
-PROG uprobe:./testprogs/uprobe_test:uprobeFunction2 { printf("foo1->a = %d\n", args.foo1->a); exit(); }
-EXPECT foo1->a = 123
+PROG uprobe:./testprogs/uprobe_test:uprobeFunction2 { printf("foo1.a = %d\n", args.foo1.a); exit(); }
+EXPECT foo1.a = 123
 REQUIRES_FEATURE dwarf
 TIMEOUT 5
 BEFORE ./testprogs/uprobe_test
 
 # Checking backwards compatibility
 NAME uprobe args as pointer
-PROG uprobe:./testprogs/uprobe_test:uprobeFunction1 { printf("c = %c\n", args->c); exit(); }
+PROG uprobe:./testprogs/uprobe_test:uprobeFunction1 { printf("c = %c\n", args.c); exit(); }
```

Comment:
This one should use `->` since it's explicitly checking for backwards compatibility.

Reply:
Done.